From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 29 Oct 2018 01:50:55 +0000
Subject: power: Avoid ABI change in 4.9.131
Forwarded: not-needed

Commit 3ba28752cca7 "power: remove possible deadlock when
unregistering power_supply" added new member power_supply::removing.
This is ABI compatible because it just takes up a padding byte, but we
need to hide the change from genksyms.

---
--- a/include/linux/power_supply.h
+++ b/include/linux/power_supply.h
@@ -249,7 +249,10 @@ struct power_supply {
 	spinlock_t changed_lock;
 	bool changed;
 	bool initialized;
+#ifndef __GENKSYMS__
 	bool removing;
+	/* 1 padding byte spare */
+#endif
 	atomic_t use_cnt;
 #ifdef CONFIG_THERMAL
 	struct thermal_zone_device *tzd;
